scripts/sandbox_start.sh

```diff
@@ -15,7 +15,8 @@ setupFabric() {
 # TODO we need health check for peer
 sleep 20
 cd test/bdd && go test
-curl -d '{"name":"demo", "did":"did:demo:abc", "uri":"http://demo.com", "signatureType":"Ed25519Signature2018", "creator":"did:demo:abc#key1" }' -H "Content-Type: application/json" -X POST http://localhost:8070/profile
+curl -d '{"name":"sidetree", "uri":"http://demo.com", "signatureType":"Ed25519Signature2018"}' -H "Content-Type: application/json" -X POST http://localhost:8070/profile
+curl -d '{"name":"veres", "uri":"http://demo.com", "signatureType":"Ed25519Signature2018", "DID":"did:v1:test:nym:z6MkhLbRigh9utJNCaiEAdkqktz4r7yVBFDeaeqCeT7pRFnF","DIDPrivateKey":"5dF8yAW7hjLkJsfMXKqTPdDZUT56dX7Jq7TdXEtUEHHt2YUFAE34nQwyPCEp5XdWCKPSxs69xXqozsNh6MoJTmz5"}' -H "Content-Type: application/json" -X POST http://localhost8070/profile
```

Comment:
Usually profiles will have different URI, you can have something like vc-issuer-1.com and vc-issuer-2.com, Also I would not use names sidetree and veres for profile names.

Reply:
done